### PR TITLE
DIT-2442 | Queries for Assigned to me cases

### PR DESCRIPTION
**Pull request by gorkemunsal**

### CODE REVIEW

File: app/controllers/v2/MyCasesController.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                                   val commonCasesView: views.html.v2.common_cases_view,
+                                   implicit val appConfig: AppConfig
+                                 ) extends FrontendController(mcc)
+  with I18nSupport {
+
+  def displayMyCases(activeSubNav: SubNavigationTab = AssignedToMeTab): Action[AnyContent] =
+    (verify.authenticated andThen verify.mustHave(Permission.VIEW_MY_CASES)).async { implicit request: AuthenticatedRequest[AnyContent] =>
+
+      for {
+        cases <- casesService.getCasesByAssignee(request.operator, NoPagination())
```
Comment (DavidGregory084) [inline app/controllers/v2/MyCasesController.scala:45]: Should this use pagination? Or is that another ticket?

File: app/views/v2/common_cases_view.scala.html [hunk 2]
```diff
@@ -18,78 +18,23 @@
...
 @import views.html.components.primary_navigation
-@import views.html.components.common_correspondence
-@import views.html.components.common_miscellaneous
+@import models.viewmodels.{ApplicationTabViewModel, MyCasesTab}
+
 
 @this(my_cases_secondary_nav: views.html.components.my_cases_secondary_navigation)
 
-@(title: String)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+@(myCases : ApplicationTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
```
Comment (DavidGregory084) [inline app/views/v2/common_cases_view.scala.html:28]: Should this be called my_cases_view now?

File: app/views/v2/common_all_open_cases_view.scala.html [hunk 3]
```diff
@@ -23,9 +23,9 @@
 
 @this()
 
-@(title: String, cases : CasesTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+@(cases : CasesTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
```
Comment (DavidGregory084) [inline app/views/v2/common_all_open_cases_view.scala.html:26]: Could this just be called "open_cases_view"? Or is it common to several pages?

**Review by DavidGregory084 — APPROVED** [review]
Looks good just a couple questions 👍

**Review by kuldeepwadhwa9 — APPROVED** [review]

File: app/views/v2/common_cases_view.scala.html (hunk 2 above)
Comment (periklas) [reply app/views/v2/common_cases_view.scala.html]: 👍

File: app/views/v2/common_all_open_cases_view.scala.html (hunk 3 above)
Comment (periklas) [reply app/views/v2/common_all_open_cases_view.scala.html:26]: 👍

File: app/controllers/v2/MyCasesController.scala (hunk 1 above)
Comment (periklas) [reply app/controllers/v2/MyCasesController.scala:45]: As per our conversation, pagination will be part of a new ticket.

**Pull request merged**